tools/yulPhaser/Population.h

```diff
@@ -0,0 +1,89 @@
+/*
+	This file is part of solidity.
+
+	solidity is free software: you can redistribute it and/or modify
+	it under the terms of the GNU General Public License as published by
+	the Free Software Foundation, either version 3 of the License, or
+	(at your option) any later version.
+
+	solidity is distributed in the hope that it will be useful,
+	but WITHOUT ANY WARRANTY; without even the implied warranty of
+	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+	GNU General Public License for more details.
+
+	You should have received a copy of the GNU General Public License
+	along with solidity.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#pragma once
+
+#include <tools/yulPhaser/Chromosome.h>
+#include <tools/yulPhaser/Random.h>
+
+#include <liblangutil/CharStream.h>
+
+#include <optional>
+#include <ostream>
+#include <vector>
+
+namespace solidity::phaser
+{
+
+/**
+ * Information describing the state of an individual member of the population during the course
+ * of the genetic algorithm.
+ */
+struct Individual
+{
+	Chromosome chromosome;
+	std::optional<size_t> fitness;
```

Comment:
```suggestion
	std::optional<size_t> fitness = nullopt;
```
to allow Individual being constructor from a Chromosome?

Reply:
Done.